docs/http/traffic-policy/actions/index.mdx

```diff
@@ -9,16 +9,16 @@ pagination_label: Actions
 
 Traffic Policy actions enable you to modify the behavior of traffic flowing through your endpoints. The following actions are available for use on your endpoints:
 
-| Type                                   | Description                                                           | Supported On      |
-| -------------------------------------- | --------------------------------------------------------------------- | ----------------- |
-| [add-headers](add-headers)             | Add custom headers to incoming requests or outgoing responses.        | Inbound, Outbound |
-| [compress-response](compress-response) | Compress HTTP response bodies from your upstream server.              | Inbound, Outbound |
-| [custom-response](custom-response)     | Send a predefined custom response directly to the client.             | Inbound, Outbound |
-| [deny](deny)                           | Block incoming traffic to an endpoint.                                | Inbound           |
-| [jwt-validation](jwt-validation)       | Validate JSON Web Tokens (JWT).                                       | Inbound           |
-| [log](log)                             | Add metadata to events for logging and monitoring.                    | Inbound, Outbound |
-| [rate-limit](rate-limit)               | Rate limit traffic to your upstream servers.                          | Inbound           |
-| [redirect](redirect)                   | Redirect users through URL transformations using regular expressions. | Inbound           |
-| [remove-headers](remove-headers)       | Remove headers from incoming requests or outgoing responses.          | Inbound, Outbound |
-| [restrict-ips](restrict-ips)           | Allow or deny traffic based on source IP.                             | Inbound           |
-| [url-rewrite](url-rewrite)             | Rewrite request URLs transparently using regular expressions.         | Inbound           |
+| Type                                   | Description                                                           | Supported On                      |
+| -------------------------------------- | --------------------------------------------------------------------- | --------------------------------- |
+| [add-headers](add-headers)             | Add custom headers to incoming requests or outgoing responses.        | on_http_request, on_http_response |
+| [compress-response](compress-response) | Compress HTTP response bodies from your upstream server.              | on_http_request, on_http_response |
+| [custom-response](custom-response)     | Send a predefined custom response directly to the client.             | on_http_request, on_http_response |
+| [deny](deny)                           | Block incoming traffic to an endpoint.                                | on_http_request                   |
```

Comment:
this is available in on_tcp_connect, right?

Reply:
yes!
We discussed this in the morning meeting, we are going to link to the tcp docs from within the http page. 
This is a "hack" just to point people in the right direction without doing a docs overhaul. at some point, docs will be overhauled.